content/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners.md

```diff
@@ -39,6 +39,12 @@ Each CODEOWNERS file assigns the code owners for a single branch in the reposito
 
 For code owners to receive review requests, the CODEOWNERS file must be on the base branch of the pull request. For example, if you assign `@octocat` as the code owner for *.js* files on the `gh-pages` branch of your repository, `@octocat` will receive review requests when a pull request with changes to *.js* files is opened between the head branch and `gh-pages`.
 
+## CODEOWNERS file size
+
+CODEOWNERS files must be under 3 MB in size. A CODEOWNERS file over this limit will not be loaded, causing code owner information not to be shown and for the appropriate code owners not to be requested to review changes in a pull request.
```

Comment:
Think we need to add a conditional so the docs are accurate for GHES and GHAE since this change won't land until GHES 3.3. For example:

```
{% ifversion fpt or ghae or ghes >= 3.3 %}
```

(you may want to check with the docs team or see if there's a common pattern for doing this)